modules/core/src/main/java/org/apache/ignite/internal/processors/datastreamer/DataStreamerImpl.java

```diff
@@ -1423,6 +1434,45 @@ private void ensureCacheStarted() {
         }
     }
 
+    /**
+     *
+     */
+    private class ThreadBuffer {
+        /** Entries. */
+        private final List<DataStreamerEntry> entries;
+
+        /** Future. */
+        private final IgniteCacheFutureImpl fut;
+
+        /**
+         * @param fut Future.
+         * @param entries Entries.
+         */
+        private ThreadBuffer(IgniteCacheFutureImpl fut, List<DataStreamerEntry> entries) {
+            assert fut != null;
```

Comment:
Compact similar semantic units like assert and setting field value. Redundant blank lines here.